README.md

```diff
@@ -301,6 +301,43 @@ function FeatureComponent() {
 }
 ```
 
+
+## `useFeature` Hook
+
+A [React Hook](https://reactjs.org/docs/hooks-intro.html) to retrieve the status of a feature flag and its variables. This can be useful as an alternative to the `<OptimizelyFeature>` component or to use features & variables inside code that is not explicitly rendered.
+
+*arguments*
+* `feature : string` Key of the feature
+* `options : Object`
+  * `autoUpdate : boolean` (optional) If true, this component will re-render in response to datafile or user changes. Default: `false`.
+  * `timeout : number` (optional) Rendering timeout as described in the `OptimizelyProvider` section. Overrides any timeout set on the ancestor `OptimizelyProvider`. 
+* `overrides : Object`
+  * `overrideUserId : string` (optional) Override the userId for calls to `isFeatureEnabled` for this hook.
+  * `overrideAttributes : optimizely.UserAttributes` (optional) Override the user attributes for calls to `isFeatureEnabled` for this hook.
+
+### Render something if feature is enabled
+
+```jsx
+import { useEffect } from 'react';
+import { useFeature } from '@optimizely/react-sdk';
+
+function LoginComponent() {
+  const [isEnabled, variables] = useFeature('feature1', { autoUpdate: true }, { /* (Optional) User overrides */ });
+  useEffect(() => {
+    document.title = isEnabled ? 'login1' : 'login2';
+  }, [isEnabled]);
+
+  return (
+    <p>
+      <a href={isEnabled ? "/login" : "/login2"}>
+        {variables.loginText}
+      </a>
+    </p>
+  )
+}
+```
+

```

Comment:
We should document in here the extra things the hook provides: `clientReady`, `didTimeout`